sphinx/domains/python.py

```diff
@@ -446,6 +446,25 @@ def get_index_text(self, modname: str, name_cls: Tuple[str, str]) -> str:
 class PyVariable(PyObject):
     """Description of a variable."""
 
+    option_spec = PyObject.option_spec.copy()
+    option_spec.update({
+        'type': directives.unchanged,
+        'value': directives.unchanged,
+    })
+
+    def handle_signature(self, sig: str, signode: desc_signature) -> Tuple[str, str]:
+        fullname, prefix = super().handle_signature(sig, signode)
+
+        typ = self.options.get('type')
+        if typ:
+            signode += addnodes.desc_annotation(typ, ': ' + typ)
```

Comment:
I'd like to add a new node for this purpose. But it will bring incompatibility. So I keep it as is.
In another usage, we can split "type" value itself and markers like this:
```
signode += addnodes.desc_annotation(typ, typ, prefix=': ')
```
I think it is much better. But it is also incompatible change...

Reply:
Okay, let's keep as is.